gradle/versions.gradle

```diff
@@ -12,6 +12,9 @@ ext {
     androidMavenGradlePluginVersion = "1.4.1"
 
     // Libraries
+    androidxTestRunnerVersion = "1.2.0"
```

Comment:
Don't forget to make corresponding update in `android-complete` :)

Reply:
(For all these deps)

Reply:
Opened up a PR here: https://github.com/AzureAD/android-complete/pull/41